veg2hab/vegkartering.py

```diff
@@ -263,14 +267,22 @@ def sorteer_vegtypeinfos_en_habkeuzes(row: gpd.GeoSeries) -> gpd.GeoSeries:
         # Er zijn geen vegtypeinfos, dus er is maar 1 habitatkeuze (H0000)
         return row
 
-    keuze_en_vegtypeinfo = list(zip(row["HabitatKeuze"], row["VegTypeInfo"]))
+    keuze_vegtypeinfo_en_voorstellen = list(
+        zip(row["HabitatKeuze"], row["VegTypeInfo"], row["HabitatVoorstel"])
+    )
     # Sorteer op basis van de habitatkeuze (idx 0)
-    sorted_keuze_en_vegtypeinfo = sorted(keuze_en_vegtypeinfo, key=rank_habitatkeuzes)
+    sorted_keuze_vegtypeinfo_en_voorstellen = sorted(
+        keuze_vegtypeinfo_en_voorstellen, key=rank_habitatkeuzes
+    )
 
-    row["HabitatKeuze"], row["VegTypeInfo"] = zip(*sorted_keuze_en_vegtypeinfo)
+    row["HabitatKeuze"], row["VegTypeInfo"], row["HabitatVoorstel"] = zip(
+        *sorted_keuze_vegtypeinfo_en_voorstellen
+    )
     # Tuples uit zip omzetten naar lists
-    row["HabitatKeuze"], row["VegTypeInfo"] = list(row["HabitatKeuze"]), list(
-        row["VegTypeInfo"]
+    row["HabitatKeuze"], row["VegTypeInfo"], row["HabitatVoorstel"] = (
+        list(row["HabitatKeuze"]),
+        list(row["VegTypeInfo"]),
+        list(row["HabitatVoorstel"]),
```

Comment:
Deze implementatie was me nog niet zo opgevallen. Bij mij komt meteen argsort naar boven als optie: Zie https://numpy.org/doc/stable/reference/generated/numpy.argsort.html, al heeft die helaas geen implementatie in default python, dus dit vind ik ook wel leuk!

Reply:
Zowel de keuze als de vegtypeinfo zijn nodig voor het bepalen van de volgorde, dus om het op deze manier te doen was heel logisch, en voor het gemak heb ik de voorstellen er gewoon aan toegevoegd. Ik heb nog wel even gedacht aan iets als argsort maar het leek me niet zo heel nodig.